### ISSUE TITLE
Extensively test ns

### ISSUE THREAD

**Issue by arichiardi** [ScalaConsultants/replumb#62]
Given that it is one of the most important functions and differs a bit from Clojure, it would be great to enhance the tests currently in `repl-test`.

The ClojureScript's [wiki page](https://github.com/clojure/clojurescript/wiki/Differences-from-Clojure#namespaces) can be taken as guide.

**Comment by tomasz-biernacki** [comment]
@arichiardi [here](https://github.com/clojure/clojurescript/wiki/Differences-from-Clojure#other-libraries) I see in the list of ported namespaces `clojure.data`
*lein repl*
```
user=> (require 'clojure.data)
nil
user=> (clojure.data/diff '(1 2 3 6) '(3 4 5 6))
[[1 2 3] [3 4 5] [nil nil nil 6]]
```
*lein fig-dev**
```
cljs.user=> (require 'clojure.data)
nil
cljs.user=>  (clojure.data/diff '(1 2 3 6) '(3 4 5 6))
ERROR - Cannot read property 'diff' of undefined
```

**Comment by tomasz-biernacki** [comment]
Also `clojure.zip`
Actually, looking in `compiled/out/clojure` there are only `set` `string` and `walk`

**Comment by arichiardi** [comment]
Yes this is normal behavior. We depend on `out` for demo repls. Actually `replumb` clients can specify anything in `:src-paths` so a user can bundle all the libs she wants. We delegate the responsibility to client code.
Again, this needs clarification and I will write a wiki page on it.

**Comment by arichiardi** [comment]
I had a newbie doubt on this but [this](https://stackoverflow.com/questions/15939816/how-to-output-individual-files-to-a-specified-directory-for-cljsbuild/28262024#28262024) and [this](https://github.com/clojure/clojurescript/wiki/Compiler-Options#output-dir) make me think that in `:output-dir` we have just what we are actually using in our codebase, so in order to have every namespace available we would probably need to bundle it (#69).

**Comment by arichiardi** [comment]
A bit of a joke, @tomasz-biernacki  you chose the only synonym of fix that is not understood by [github](https://help.github.com/articles/closing-issues-via-commit-messages/).

Closed by #75.

**Comment by tomasz-biernacki** [comment]
True...
